packages/beacon-node/src/chain/prepareNextSlot.ts

```diff
@@ -104,6 +104,12 @@ export class PrepareNextSlotScheduler {
         RegenCaller.precomputeEpoch
       );
 
+      // cache HashObjects for faster hashTreeRoot() later, especially for computeNewStateRoot() if we need to produce a block at slot 0 of epoch
+      // see https://github.com/ChainSafe/lodestar/issues/6194
+      const hashTreeRootTimer = this.metrics?.stateHashTreeRootTime.startTimer({source: "prepare_next_slot"});
```

Comment:
What should be the naming convention we use for source?

e.g. here we use camelCase (function names)
https://github.com/ChainSafe/lodestar/blob/465834846a18619a30d7b0e11dfcac512248c0ec/packages/state-transition/src/stateTransition.ts#L61

and recently added (https://github.com/ChainSafe/lodestar/pull/6143) step metrics also use camelCase
https://github.com/ChainSafe/lodestar/blob/465834846a18619a30d7b0e11dfcac512248c0ec/packages/state-transition/src/stateTransition.ts#L183

I kinda like the function names and might be better for consistency, but maybe not always applicable (i.e. if there is no function call specific to the metric)

Reply:
> What should be the naming convention we use for source?

agree to have some naming convention, we used snake case as suggested by @dapplion and that's my preference. Agree there are consistencies so need some agreement to work on

I don't want to use function names, just which whatever name that makes sense in the context because:
- some function names are so long so it's not nice to render in grafana, for example `verifyBlocksStateTransitionOnly`
- one function could call a metric multiple times

so this one should be more dynamic to me, as long as a PR gets through review process we're fine I think

Reply:
> I don't want to use function names, just which whatever name that makes sense in the context because:

Makes sense to not be to strict here then, also I guess those labels are mostly used to visualize and separate metric values in a panel. As long as it is visually readable and makes sense in the context of the metrics the naming convention (camelCase vs snake_case) should not matter that much.

Reply:
snake case just reads better in grafana :)